src/commands.py

```diff
@@ -388,7 +388,7 @@ def force_ticks(
         author: Union[AccountId, str],
         amount: int, server: Server):
     """Forcibly run multiple ticks"""
-    author_acc = get_account(author, server)
+    author_acc = _get_account(author, server)
```

Comment:
Still a bit iffy on the author_acc thing. no function on the server takes in an AccountId, therefore it shouldn't be necessary here to keep the IDs

Reply:
the author is almost always taken as an account id by the server